vscode/src/circuit.ts

```diff
@@ -219,7 +243,7 @@ function documentHtml(maybeUri: string, range?: IRange) {
   return location;
 }
 
-function escapeHtml(unsafe: string): string {
+export function escapeHtml(unsafe: string): string {
```

Comment:
I see in session.ts you now use `import { escapeHtml } from "markdown-it/lib/common/utils";`. Is that usable here also?

Reply:
Wow, I had no idea - auto-import must have added that. I might as well use it here too then :)